File: 1848-sum-of-unique-elements/sum-of-unique-elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
        HashMap<Integer,Integer> freq = new HashMap<>();

        for(int i:nums){ 
            freq.putIfAbsent(i,0);
            freq.put(i,freq.get(i)+1);
        }
        
        int sum = 0;

        for(Map.Entry<Integer,Integer> entry : freq.entrySet()){
            if(entry.getValue() == 1)sum+=entry.getKey();
        }
        return sum;

            
    }
}

File: 1848-sum-of-unique-elements/sum-of-unique-elements.py
class Solution(object):
    def sumOfUnique(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        freq = {}
        for i in nums: 
            if i in freq: 
                freq[i] += 1
            else: 
                freq[i] = 1
        sum = 0
        for value,freq in freq.items():
            if(freq==1):
                sum+=value

        return sum
            
            
        
        